dt-core/admin/menu/tabs/tab-custom-fields.php

```diff
@@ -401,21 +401,23 @@ private function edit_field( $field_key, $post_type ){
                         </div>
                     </td>
                 </tr>
-                <?php if ( isset( $field["icon"] ) && ! empty( $field["icon"] ) ): ?>
+                <?php if ( ( isset( $field["icon"] ) && ! empty( $field["icon"] ) ) || isset( $field["customizable"] ) ): ?>
```

Comment:
@kodinkat, even if the icon param isn't set? Can we allow the user to set an icon?